lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -225,42 +225,19 @@ def get_zaxis(self):
     get_zgridlines = _axis_method_wrapper("zaxis", "get_gridlines")
     get_zticklines = _axis_method_wrapper("zaxis", "get_ticklines")
 
-    def _unit_cube(self, vals=None):
+    def _tunit_cube(self, vals, M):
```

Comment:
Could you add a comment describing the function? That this returns a transformed unit cube based on the projection matrix M

Reply:
(It took me until now to grok that "tunit" was short for "transformed unit")

Reply:
I renamed it to `_transformed_cube` and added a minor doc-string. Also, after reading the calling code a bit more carefully, I also removed the M-argument since that is always `self.M` anyway.